Update for titles and short stories on assemblies (#108230)
openshift/openshift-docs: virt/managing_vms/virt-migrating-vms-in-single-cluster-to-different-storage-class.adoc

:_mod-docs-content-type: ASSEMBLY
[id="virt-migrating-vms-in-single-cluster-to-different-storage-class"]
= Migrate VMs in a single cluster to a different storage class
include::_attributes/common-attributes.adoc[]
:context: virt-migrating-vms-in-single-cluster-to-different-storage-class

toc::[]

[role="_abstract"]

> Migrate virtual machines (VMs) within a single cluster from one storage class to a different storage class to optimize for storage performance or cost. By using the {product-title} web console, you can perform the migration for the VMs in bulk.

Reviewer: I like that you are adding the "Why" here, but I think the first sentence is a bit too dense with information to be easily parsed. Here's a possible suggestion:

```suggestion
To optimize performance or reduce costs, you can migrate virtual machines (VMs) between storage classes within a single cluster. You can use the {product-title} web console to perform these migrations in bulk.
```
